### ISSUE TITLE
🐛 Clarify where settings.php files are located

### ISSUE THREAD

**Issue by hktang** [platformsh/platformsh-docs#4133]
### Where on docs.platform.sh should be changed?

https://docs.platform.sh/guides/drupal/deploy/customize.html#settingsphp

### What exactly should be updated?

Hi, I am following the Drupal guide trying to wire up an existing Drupal project. I *think* the settings.php file here refers to the one located under `./web/sites/default/` but I am not 100% sure. 

This confusion is complicated with my existing project containing a settings.pantheon.php at the project root, and an example.settings.local.php in the parent of sites/default, i.e. `./web/sites/`. 

Looking at the platform.sh d10 template, I think I should to put _settings_, _settings.platformsh_ under `./web/sites/default`. And, this line means I should also put the settings.local file inside `./web/sites/default` and not `./web/sites/`, right?

https://github.com/platformsh-templates/drupal10/blob/cebfa9f2fbf6e889dd9af9ccf97fc48fd24dcbde/web/sites/default/settings.php#L37

In summary, it would be nice to provide a clear definition of where each settings file should ideally be located, to give a newbie like myself peace of mind. 

Thank you for considering!
